boards/arm/mps2/CMakeLists.txt

```diff
@@ -21,9 +16,8 @@ if(CONFIG_SOC_MPS2_AN521_CPU1 AND NOT CONFIG_OPENAMP)
     empty_cpu0
     SOURCE_DIR ${BOARD_DIR}/empty_cpu0
     INSTALL_COMMAND  ""
-    CMAKE_CACHE_ARGS -DBOARD:STRING=mps2_an521
+    CMAKE_CACHE_ARGS -DBOARD:STRING=mps2/an521/cpu0
```

Comment:
in relation to the comment related to soc vs board testing, should we use:
```
CMAKE_CACHE_ARGS -DBOARD:STRING=${BOARD}/an521/cpu0
```

That would allow copying code / change board name as long as soc stays the same.

Reply:
Fixed